### PR TITLE
sunburst

### PR DESCRIPTION
**Pull request by PythonGurl**
Can you help check the graph on graphs.ipynb? 
It doesn't show up.

### CODE REVIEW

**Comment by nerner-code** [comment]
Can you give me write access to this repo?  I have a notebook with a working sunburst chart, but I can't push it because I don't have write access.

**Comment by PythonGurl** [comment]
image

File: app.py [hunk 1]
```diff
@@ -18,31 +18,43 @@
...
+    inventory_df = read_data.get_rental_inventory()
     inventory_graph = [
         dcc.Dropdown(sorted(boroughs.keys()),
                      placeholder='Select a borough', id='borough-dropdown'),
         dcc.Dropdown(id='areas-dropdown', placeholder='Areas'),
-        dcc.Graph(id='area-inventory')
+        dcc.Graph(id='area-inventory-line')
+    ]
+    inventory_sunburst = [
+        dcc.Dropdown(sorted(list(set(inventory_df['date'].dt.year))), id='year-dropdown',
```
Comment (nerner-code) [inline app.py:29]: Calling `sorted` on an iterable in python will return a list.  So, casting your set to a list here is redundant.

File: app.py [hunk 2]
```diff
@@ -18,31 +18,43 @@
...
+                dbc.Tab(inventory_graph, label='Rental Listings by Borough'),
+                dbc.Tab(inventory_sunburst, label='Rental Listings by Year')
             ]
         ),
     ])
 
 
 app.layout = get_layout()
 
+# callbacks ---------------------------------------------
```
Comment (nerner-code) [inline app.py:47]: Did you add these comments for documentation?  You could probably just give your callbacks descriptive names so we can tell what they are doing.

Comment (PythonGurl) [reply app.py:47]: yes, I added it. it just means that below is a callback section.

**Review by nerner-code — APPROVED** [review]

**Pull request merged**